src/net.cpp

```diff
@@ -229,8 +229,9 @@ void AdvertiseLocal(CNode* pnode)
             addrLocal.SetIP(pnode->addrLocal);
         }
         if (addrLocal.IsRoutable()) {
-            LogPrintf("AdvertiseLocal: advertising address %s\n", addrLocal.ToString());
-            pnode->PushAddress(addrLocal);
+            LogPrintf("AdvertizeLocal: advertising address %s\n", addrLocal.ToString());
```

Comment:
nit: fix spelling of function name `AdvertiseLocal` (or just use `__func__`)

Reply:
The function at the time this was submitted was misspelled.   https://github.com/PIVX-Project/PIVX/pull/951 corrected it; so this will clear up with a rebase.

Reply:
Doublechecked #951; and yea... nice catch, looks like a rebase error (and I agree, I should have changed to `__func__`)